### PR TITLE
testing linear regime

### PR DESCRIPTION
**Pull request by pzhokhov**

### CODE REVIEW

**Review by sashaaz — APPROVED** [review]
I love it its great keep.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*.o
+*.bin
+*.udf
+bin/*
```
Comment (sashaaz) [inline .gitignore]: You don't want to add your beautiful data?

File: Makefile [hunk 2]
```diff
@@ -29,6 +29,8 @@ gsolver_solid_qdht : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D
 gsolver_solid_ux : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_MEASURE -DTRANSVERSE_DIMENSIONS=1
 gsolver_solid_1d : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_MEASURE -DNO_DIFFRACTION
 gsolver_solid : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_MEASURE -DTRANSVERSE_DIMENSIONS=2
+gsolver_solid_qdht : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_ESTIMATE -DTRANSVERSE_DIMENSIONS=1 -DQDHT -D_UNIAXIAL
```
Comment (sashaaz) [inline Makefile:32]: Why change the flags?

File: Makefile [hunk 3]
```diff
@@ -29,6 +29,8 @@ gsolver_solid_qdht : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D
...
+
```
Comment (sashaaz) [inline Makefile:33]: This is just you adding a line right?

File: Makefile (hunk 2 above)
Comment (pzhokhov) [reply Makefile:32]: Which one? If you are talking about fftw flags - to make code start faster (measure vs estimate don't matter much in short term, to my experience). The rest of them - to make a axisymmetric solver with Keldysh ionization

File: Makefile (hunk 3 above)
Comment (pzhokhov) [reply Makefile:33]: Yeah

**Pull request merged**